src/Components/Users_List.jsx

```diff
@@ -0,0 +1,28 @@
+import React, {useState, useEffect} from 'react';
+import UsersChart from './Users_Chart';
+import styles from './css/style.css';
+
+const Users_List = (props) => {
+    const user = props.userList;

```

Comment:
Naming suggestion: naming a variable as `user` one is thinking you are referring to a single user whereas you are referring here to a list of users.